ls1intum/Artemis
# `Exam mode`: Validate that exam exercises do not have dates

### Checklist
#### General


#### Server
#### Changes affecting Programming Exercises

### Motivation and Context



### Description


### Steps for Testing

1 Editor or higher

The bugfix is validated by introduced tests, only adaptions for the Resource should be tested

Create and Update a Modeling Exercise, first in a Course, then in an Exam

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots
